app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Good job implementing the authentication however your login and signup pages should look like the provided wireframes, kindly make sure to implement this so your project is completed

||||
|-|-|-|
|![Screenshot_20230202_172126](https://user-images.githubusercontent.com/25227022/216382947-55c31511-386e-486a-9830-dd0fd9cdbe42.png)|![Screenshot_20230202_172131](https://user-images.githubusercontent.com/25227022/216382970-2d12a663-081b-4512-80dc-fb5789257d5b.png)|![image](https://user-images.githubusercontent.com/25227022/216383255-33d4fd58-29e6-417b-9b68-f97b4f6597dd.png)|